lib/nodejs/vwf.js

```diff
@@ -7,7 +7,9 @@ var parseurl = require( './parse-url' ),
     servehandler = require( './serve-handler' ),
     helpers = require( './helpers' ),
     application = require( './application' ),
-    url = require( 'url' );
+    url = require( 'url' ),
+    lobby = require( './lobby' ),
+    finalhandler = require( 'express/node_modules/finalhandler' );
```

Comment:
Um, no. I guess I really wanted to be _just like Express_. But that's weird, especially since this file doesn't even reference Express directly. Fixed.